Tests/Remora.Discord.Tests/Samples/Gateway/Events/GUILD_UPDATE/2021-06-17 05.46.43Z.json

```diff
@@ -0,0 +1,332 @@
+        "permissions": "6513086145",
+        "name": "pingabletest",
+        "mentionable": true,
+        "managed": false,
+        "id": "565985171556270159",
+        "hoist": false,
+        "color": 0
+      },
+      {
+        "position": 9,
+        "permissions": "6513086145",
+        "name": "with space",
+        "mentionable": false,
+        "managed": false,
+        "id": "566557912445485061",
+        "hoist": false,
+        "color": 0
+      },
+      {
+        "position": 8,
+        "permissions": "6513086145",
+        "name": "unmentionable",
+        "mentionable": false,
+        "managed": false,
+        "id": "573001248043696141",
+        "hoist": false,
+        "color": 0
+      },
+      {
+        "position": 6,
+        "permissions": "6513086145",
+        "name": "insane",
+        "mentionable": false,
+        "managed": false,
+        "id": "616312641538883605",
+        "hoist": false,
+        "color": 0
+      },
+      {
+        "position": 5,
+        "permissions": "6513086145",
+        "name": "new role",
+        "mentionable": false,
+        "managed": false,
+        "id": "666411368102232065",
+        "hoist": false,
+        "color": 0
+      },
+      {
+        "position": 4,
+        "permissions": "6513086145",
+        "name": "Jays-BB-8 bot",
+        "mentionable": false,
+        "managed": false,
+        "id": "676995239570964490",
+        "hoist": false,
+        "color": 15158332
+      },
+      {
+        "position": 3,
+        "permissions": "6513086145",
+        "name": "Its The Best Rank Ever",
+        "mentionable": false,
+        "managed": false,
+        "id": "676997069055197216",
+        "hoist": false,
+        "color": 3066993
+      },
+      {
+        "tags": { "bot_id": "711272498100174960" },
+        "position": 2,
+        "permissions": "7785016400",
+        "name": "Modix-Translation-Dev",
+        "mentionable": false,
+        "managed": true,
+        "id": "711423124314521601",
+        "hoist": false,
+        "color": 0
+      }
+    ],
+    "splash": null,
+    "afk_timeout": 300,
+    "vanity_url_code": null,
+    "icon": "49fab25a1850a96ba99459635fdbf42b",
+    "emojis": [
+      {
+        "roles": [],
+        "require_colons": true,
+        "name": "tldr",
+        "managed": false,
+        "id": "521568824038916107",
+        "available": true,
+        "animated": false
+      },
+      {
+        "roles": [],
+        "require_colons": true,
+        "name": "modix",
+        "managed": false,
+        "id": "559696137171501066",
+        "available": true,
+        "animated": false
+      },
+      {
+        "roles": [],
+        "require_colons": true,
+        "name": "GBlobShuffle",
+        "managed": false,
+        "id": "570344299846434817",
+        "available": true,
+        "animated": true
+      }
+    ],
+    "max_video_channel_users": 25,
+    "preferred_locale": "en-US",
+    "region": "us-east",
```

Comment:
See the previous notes about "region" being deprecated.

Reply:
This is a direct sample from Discord. Deprecated or not, the field is sent by Discord. I've added it to the ignored fields list in tests to account for that.

Reply:
Fair enough.